src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -98,6 +108,12 @@ private enum Mode
         public string FallbackHref { get; set; }
 
         /// <summary>
+        /// Value indicating if file version should be appended to the href urls.
+        /// </summary>
+        [HtmlAttributeName(FileVersionAttributeName)]
+        public string FileVersion { get; set; }
```

Comment:
@DamianEdwards might be more HTML-ish if we converted it to a bool using `var addFileVersion = FileVersion != null;`.  is that better or worse?